omnibus/resources/agent/msi/source.wxs.erb

```diff
@@ -344,6 +344,16 @@
         </Component>
       </Directory>
 
+      <Directory Id="run" Name="run">
+        <Component Id="run"
+                   Guid="5acf7ec1-3bcf-40d7-8840-f8de8a472c3c"
+                   Permanent="yes"
+                   Location="either"
+                   NeverOverwrite="yes">
+          <CreateFolder/>
+        </Component>
+      </Directory>
+
```

Comment:
Argh.  Github won't let me put in a marker where you need this.  Below, line ~434, you need an entry to actually create this directory like 
`<ComponentRef Id="run" />`

Also, a more descriptive name/id would be nice, and/or a comment as to what this is for.